src/directives/mmsDiffAttr.js

```diff
@@ -17,212 +17,166 @@ angular.module('mms.directives')
  * @description
  *  Compares a element at two different refs/commits and generates a pretty diff.
  * ## Example
- * <mms-diff-attr mms-base-element-id="" (mms-compare-element-id="") mms-attr="name|doc|val"
- * (mms-base-project-id="" mms-compare-project-id="" mms-base-ref-id="" mms-compare-ref-id=""
+ * <mms-diff-attr mms-element-id="" mms-attr="name|doc|val"
+ * (mms-project-id="" mms-base-ref-id="" mms-compare-ref-id=""
  * mms-base-commit-id="" mms-compare-commit-id="")></mms-diff-attr>
- *
- * @param {string} mmsBaseElementId The id of the element to do comparison of
```

Comment:
reason why you removed the ng-doc comments?

Reply:
oh i deleted them because they are outdated.